htgr/triso_fuel/triso_1d.i

```diff
@@ -179,6 +177,16 @@ coordinates5 = '${fparse coordinates4+OPyC_thickness}'
   []
 []
 
+[Contact]
+  [mechanical]
+    primary = IPyC_inner_boundary
+    secondary = buffer_outer_boundary
+    penalty = 1e16
+    model = frictionless
+    formulation = kinematic
+  []
```

Comment:
should this be added on every interface?

Reply:
No, only those two layers have a gap.

Reply:
I decided to remove the contact since this is not necessary for this 1D case. I will add them back if needed later.